spec/recipes_spec/books/pl-u-physics/expected_output.xhtml

```diff
@@ -33320,9 +33309,419 @@ podstaw podane w tre&#x15B;ci zadania warto&#x15B;ci drogi i czasu:<span data-ty
+                            </m:mtable>
+                          </m:annotation-xml>
+                        </m:semantics>
+                      </m:math>
+                    </div>
+                    <span data-type="media" id="auto_042cb2d3-0dd8-4809-96ce-163f2426bae6_fs-id1167131327810" data-alt="Na rysunku przedstawiono dwa dwuwymiarowe uk&#x142;ady wsp&#xF3;&#x142;rz&#x119;dnych kartezja&#x144;skich. Czerwony uk&#x142;ad wsp&#xF3;&#x142;rz&#x119;dnych S ma poziom&#x105; o&#x15B; x o zwrocie w prawo i pionow&#x105; o&#x15B; y o zwrocie w g&#xF3;r&#x119;. Niebieski uk&#x142;ad wsp&#xF3;&#x142;rz&#x119;dnych S prim ma ten sam punkt pocz&#x105;tkowy co uk&#x142;ad S, ale jest obr&#xF3;cony przeciwnie do ruchu wskaz&#xF3;wek zegara o k&#x105;t fi. W uk&#x142;adach znajduj&#x105; si&#x119; dwa punkty &#x2013; P oraz Q. Wsp&#xF3;&#x142;rz&#x119;dna x punktu P w uk&#x142;adzie S zaznaczona jest poprzez poprowadzenie od punktu linii przerywanej, r&#xF3;wnoleg&#x142;ej do osi y. Wsp&#xF3;&#x142;rz&#x119;dna y punktu P w uk&#x142;adzie S zaznaczona jest poprzez poprowadzenie od punktu linii przerywanej, r&#xF3;wnoleg&#x142;ej do osi x. Wsp&#xF3;&#x142;rz&#x119;dna x prim punktu P w uk&#x142;adzie S prim zaznaczona jest poprzez poprowadzenie od punktu linii przerywanej, r&#xF3;wnoleg&#x142;ej do osi y prim. Wsp&#xF3;&#x142;rz&#x119;dna y prim punktu P w uk&#x142;adzie S prim zaznaczona jest poprzez poprowadzenie od punktu linii przerywanej, r&#xF3;wnoleg&#x142;ej do osi x prim.">
+                      <img src="../resources/7cab7d3d301327999cd7e506d310ee90dcd7660e" data-media-type="image/jpeg" alt="Na rysunku przedstawiono dwa dwuwymiarowe uk&#x142;ady wsp&#xF3;&#x142;rz&#x119;dnych kartezja&#x144;skich. Czerwony uk&#x142;ad wsp&#xF3;&#x142;rz&#x119;dnych S ma poziom&#x105; o&#x15B; x o zwrocie w prawo i pionow&#x105; o&#x15B; y o zwrocie w g&#xF3;r&#x119;. Niebieski uk&#x142;ad wsp&#xF3;&#x142;rz&#x119;dnych S prim ma ten sam punkt pocz&#x105;tkowy co uk&#x142;ad S, ale jest obr&#xF3;cony przeciwnie do ruchu wskaz&#xF3;wek zegara o k&#x105;t fi. W uk&#x142;adach znajduj&#x105; si&#x119; dwa punkty &#x2013; P oraz Q. Wsp&#xF3;&#x142;rz&#x119;dna x punktu P w uk&#x142;adzie S zaznaczona jest poprzez poprowadzenie od punktu linii przerywanej, r&#xF3;wnoleg&#x142;ej do osi y. Wsp&#xF3;&#x142;rz&#x119;dna y punktu P w uk&#x142;adzie S zaznaczona jest poprzez poprowadzenie od punktu linii przerywanej, r&#xF3;wnoleg&#x142;ej do osi x. Wsp&#xF3;&#x142;rz&#x119;dna x prim punktu P w uk&#x142;adzie S prim zaznaczona jest poprzez poprowadzenie od punktu linii przerywanej, r&#xF3;wnoleg&#x142;ej do osi y prim. Wsp&#xF3;&#x142;rz&#x119;dna y prim punktu P w uk&#x142;adzie S prim zaznaczona jest poprzez poprowadzenie od punktu linii przerywanej, r&#xF3;wnoleg&#x142;ej do osi x prim." id="auto_042cb2d3-0dd8-4809-96ce-163f2426bae6_16" />
+                    </span>
+                  </div>
+                </div>
+              </div>
```

Comment:
Same here:
![image](https://user-images.githubusercontent.com/55203598/186692890-ec701ce7-bf41-41e7-b66c-25f2f038c42c.png)
Additional information: this is the same exercises which previously got wrapped when there were changes in `BakeFigure`.